cloudsdk/src/main/java/io/particle/android/sdk/cloud/DeviceState.java

```diff
@@ -135,6 +156,10 @@ public void writeToParcel(Parcel dest, int flags) {
         dest.writeString(deviceType.name());
         dest.writeInt(platformId);
         dest.writeInt(productId);
+        Parcelables.writeBoolean(dest, cellular);
+        dest.writeString(imei);
```

Comment:
Same thing here as elsewhere: if these are nullable, we should be using `read/writeValue()`